src/components/form/FluroContentFormField.vue

```diff
@@ -2178,20 +2175,65 @@ export default {
 		dateStringModel: {
 			get() {
 				if (this.timezone) {
-					return this.fieldModel && this.$fluro.date.moment.tz(this.fieldModel, this.timezone).format('YYYY-MM-DD');
+					// console.log("watch timezone field model", this.fieldModel)
+
+					const now = this.$fluro.date.moment(this.fieldModel);
```

Comment:
Probably want to check `this.fieldModel` first, and then you can get rid of the two checks later:
```
if (!this.fieldModel) return null;
...rest of logic
```